### ISSUE TITLE
Ignores symbol instances

### ISSUE THREAD

**Issue by Joshuwar** [awt2542/ViewController-for-Framer#15]
This could be deliberate, but I've noticed that when I create a symbol instance (in Sketch) which contains a group named with a _transition_destination_, that link is ignored (presumably because framer doesn't import the symbol as layers). Symbols given an instance name of _transition_destination_ also don't work (presumably for the same reason).

You can partly work around this by wrapping an individual symbol in a group, and naming that, but this could become hard to maintain on large, evolving prototypes. 

Any ideas?

**Comment by awt2542** [comment]
Yes, this is an issue with how Framer imports layers. Not much we can do at the moment. Let them know and maybe we'll see a change :)

**Comment by Joshuwar** [comment]
There's a bit of talk about it in the [Facebook group](https://www.facebook.com/groups/framerjs/search/?query=symbols), but I'm not sure if it's going anywhere. I might have a go at hacking the import script to deal with symbols differently, as a workaround until a decent solution is decided upon.
